Libraries/LibWeb/Credentials/Credential.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024, Miguel Sacristán <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Credential.h"
```

Comment:
Generally we still include the main file as `<LibWeb/Subpath/ClassName.h>`

Though there are plenty of exceptions to this rule if you go looking :sweat_smile: